### ISSUE TITLE
Importing an export string, causes the total counts to double (or triple, etc)

### ISSUE THREAD

**Issue by zeryl** [Raylehnhoff/HICTY-Test#1]
I exported my string from the production version of the site, and tried to import to test, and it doubled/tripled the number.  I went back to the production version, and no data was loaded, so I went ahead and imported the string I had exported minutes before, and now I show 

Armor: 9 (of 260)
Weapon: 2 (of 348)
Jewelry: 7 (of 82)
Stashed Items:109 (of 690)
Total Cubed: 18 (of 690)
Total: 127 (of 690)

These are all double what they should be (the total items, not my count)

Edit: A refresh (browser refresh) causes the numbers to be correct, so this is a display bug only, on import.

**Comment by Raylehnhoff** [comment]
Can you include the import string you used?

**Comment by zeryl** [comment]
Attached

[string.txt](https://github.com/Raylehnhoff/HICTY-Test/files/305607/string.txt)

**Comment by Raylehnhoff** [comment]
Hi @zeryl,

Good catch on this. It wasn't a display-only bug, since it was actually causing the underlying master list to duplicate itself, it's just my consistency checker removes duplicates (which runs on refresh). I believe I've fixed this issue. With your blessing on it being corrected I'll push the HICTY-Test to Live.

Sorry for the extreme delay on this; I had to travel for work last week and I've been swamped with life.
